log/lvl.go

```diff
@@ -77,108 +64,124 @@ var debugMut sync.RWMutex
 var regexpPaths, _ = regexp.Compile(".*/")
 
 func init() {
+	initStdLogger()
 	ParseEnv()
 }
 
-func lvl(lvl, skip int, args ...interface{}) {
-	debugMut.Lock()
-	defer debugMut.Unlock()
+type stdLogger struct{}
```

Comment:
Now we have three types of loggers, so I vote for putting all three of them into a `loggers.go` file separately.